KomodoPlatform/komodo-wallet-mobile
# Bug - Fix iOS ARRR Broken After App Resume

Make the ZHTLC activation process more resilient to being interrupted and for app restarts. And other bugs which were later discovered.

## Changelog:
- Fix an issue where resuming the iOS app after minimizing it would cause ARRR/ZHTLC to deactivate. The app now resyncs ARRR after minimizing.
- ARRR/ZHTLC: Hide sapling activation mode unless test coins are enabled.
- Rename some methods/variables and remove unused code.
- Minor ZHTLC UI/UX improvements.
- Set expiry for rebranding notice (November 2023).
- Implement ZHTLC coin disable/deactivation.
- Fixed pagination. Previously only 10 transactions max were shown. Now it will show as many as your fingers can scroll.
- Fix issue where disabled ZHTLC coins would re-activate after restart.
- Simplify coin details transactions list page.
- So many other improvements. See commit history

## To test (on iOS and Android):
- Test successful initial activation of the various `ARRR`/`ZOMBIE` activation modes. Testing sapling mode activation is not required.
- For past-date sync mode: After successful activation, restart the app and check that ZHTLC resyncs and transactions show.
- Check that sapling activation mode only shows when test coins mode is enabled.
- (Optional since the user is warned against it) Test what happens if the app is restarted during initial activation and during resync. We should be able to drop the warnings to the user not to do this since we can safely handle it.
